submissions/natashafir/post-oop/index.js

```diff
@@ -0,0 +1,60 @@
+class Inhabitants {
+    constructor(species, name, gender, legs, hands, saying) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.legs = legs;
+        this.hands = hands;
```

Comment:
Not all inhabitants have hands by natural design. Not that they just have `0` hands or `null` hands or `undefined` number of hands, but they do not hands property at all.
Imagine, we add birds. Under current paradigm every inhabitant would have wings, just some of them would have zero wings. This way the `Inhabitant` class becomes too general, and not easy to maintain.

The beauty of OOP is that every base class owns and handles only shared props, and every child class introduces and handles props unique to that child class.

Handling of props:
a child class can have its own method (e.g. `methodName`) where it calls parent's method (e.g. `super.methodName()`) and supplements it with some data from unique properties it **owns** directly.